.github/reviewers.yml

```diff
@@ -0,0 +1,15 @@
+reviewers:
+  defaults:
+    - richbachman
+  groups:
+    core-contributors:
+      - rssilva # cspell:disable-line
+      - RohitRajendran # cspell:disable-line
+      - BrianLincoln
+      - apvale # cspell:disable-line
+      - blumaa # cspell:disable-line
+      - jvaladas # cspell:disable-line
+
+options:
+  ignore_drafts: true
+  number_of_reviewers: 3
```

Comment:
Shouldn't we set number of approvers as well? 🤔

Reply:
But I think this is a conversation about the process in Pluto 😅  We still don't know how many approvals we want to have by PR. 🙏🏻

Reply:
At our current team size, I feel like 2 approvals will be plenty. Any design changes will also need the Chromatic approval. We can re-evaluate if we find things are slipping through.